pkg/config/config.go

```diff
@@ -598,7 +598,8 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("logs_config.stop_grace_period", 30)
 	config.BindEnvAndSetDefault("logs_config.close_timeout", 60)
 	config.BindEnvAndSetDefault("logs_config.auditor_ttl", DefaultAuditorTTL) // in hours
-	config.BindEnv("logs_config.additional_endpoints")                        //nolint:errcheck
+	config.BindEnvAndSetDefault("logs_config.remote_tagger", false)
```

Comment:
with `logs_config` which agent is it? because for me the log_agent is located in the `core-agent` that runs the `tagger server`.